tests/python/gaia-ui-tests/gaiatest/gaia_graphics_test.py

```diff
@@ -51,16 +51,12 @@ def tearDown(self):
     # if the status bar is visible, crop it off
     @property
     def crop_height(self):
-        self.marionette.switch_to_frame()
         height = 0
-        _statusbar_locator = (By.ID, 'statusbar')
-        if expected.element_displayed(*_statusbar_locator)(self.marionette):
-            # get the size of the status bar to crop off
-            status_bar = self.marionette.find_element(*System._status_bar_locator)
-            # get the size of the status bar, and multiply it by the device pixel ratio to get the exact size on device
+
+        status_bar = System(self.marionette).status_bar
+        if status_bar.is_displayed:
             height = int(status_bar.rect['height']
                                    * self.marionette.execute_script('return window.wrappedJSObject.devicePixelRatio;'))
-
         return height
```

Comment:
I think this can be replaced by:
```
status_bar = System(self.marionette).status_bar
return int(status_bar.rect['height']
                                    * self.marionette.execute_script('return window.wrappedJSObject.devicePixelRatio;'))
```
?